app/editor/src/features/content/papers/components/ReportActions.tsx

```diff
@@ -57,7 +60,7 @@ export const ReportActions: React.FunctionComponent<IReportActionProps> = ({
             : searchResults?.items.map((c) => c.id) ?? [],
         });
         if (value === 'false' || !value) {
-          removeContent(items);
+          // removeContent(items);
```

Comment:
Actually, it's good you spotted this as I commented it out because I didnt understand why it was here in the first place.  Why would a Report "Remove Content"?

Reply:
The Papers page is built to support the process of preparing the morning reports.  There are at least 3 morning reports that are fed by this page.

If I recall correctly the editor would 'add' content and update the content to align with one or more of the reports.  Adding was essentially 'publishing'.  But they do it as a group action through checkboxes.  However, as they view the 'added' content they then remove content to whittle it down to only a few key stories for each grouping.

I'd be nervous you're introducing a regression issue if there is no way to 'remove' content.  You'll need to review what 'removing' does.  I suspect it's a group action to 'unpublish' the content.

Reply:
resolved by emitting to parent component where it can be handled gracefully